calculator.go

```diff
@@ -87,7 +87,7 @@
 		if err != nil {
 			return 0, err
 		}
-		return left + right, nil
+		return left + right, err
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Returning `err` here is unnecessary as it will always be `nil`. The error check has been done just before this line and if there was an error, the function would have returned at that point.
```diff
-		return left + right, err
+		return left + right, nil
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->